pkgs/tools/networking/nxdomain/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildPythonApplication, fetchPypi, dnspython, pytestCheckHook }:
+
+buildPythonApplication rec {
+  pname = "nxdomain";
+  version = "1.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1nlzkgp87d2h5yf936h4k7grl4fg6dq1nmv5fbdv09ffd36ba10m";
+  };
+
+  propagatedBuildInputs = [ dnspython ];
+
+  checkInputs = [ pytestCheckHook ];
+
+  postCheck = ''
+    echo example.org > simple.list
+    python -m nxdomain --format dnsmasq --out dnsmasq.conf --simple ./simple.list
+    grep -q 'address=/example.org/' dnsmasq.conf
+  '';
+
+  meta = with lib; {
```

Comment:
AFAIK you should add `platforms = platforms.all;` if you want this to build on hydra

Reply:
Done, thanks.